include/class.queue.php

```diff
@@ -65,8 +70,9 @@ class CustomQueue extends VerySimpleModel {
     const FLAG_INHERIT_DEF_SORT = 0x0040; // Inherit default selected sort
     const FLAG_INHERIT_EXPORT  =  0x0080; // Inherit export fields from parent
 
+    const FLAG_VISIBLE_TEAM =     0x0100; // `staff_id` represents a team_id
 
-    const FLAG_INHERIT_EVERYTHING = 0x158; // Maskf or all INHERIT flags
+    const FLAG_INHERIT_EVERYTHING = 0x1f8; // Maskf or all INHERIT flags
```

Comment:
This was a guess. Should the `INHERIT_EVERYTHING` mask include the new `INHERIT_EXPORT` flag?